### PR TITLE
Changes to Magazine Capacity for Colt SMG and Sawed Off Shotgun

### PR DESCRIPTION
**Pull request by GuiShing**
Credit to EFdee for the Colt SMG edits.
Colt SMG mag capacity is now 32 rounds, and the Sawed-off shotgun has a mag size of 4 (based on the capacity of an Ithaca 37 riot shotgun)

### CODE REVIEW

File: System/SwatEquipment.ini [hunk 1]
```diff
@@ -2210,6 +2210,14 @@ UseAnimationRate=2.0
 AvailableFireMode=FireMode_Auto
 AvailableFireMode=FireMode_Single
 
+;; Damage
```
Comment (T-MaxWiese-T) [inline System/SwatEquipment.ini:2213]: Unnecessary

File: System/SwatEquipment.ini [hunk 2]
```diff
@@ -2244,15 +2252,19 @@ ZoomedSemiRecoilBase=209
 ZoomedBurstRecoilBase=209
 ZoomedAutoRecoilBase=180
 
-;; Variants
```
Comment (T-MaxWiese-T) [inline System/SwatEquipment.ini:2247]: Unnecessary

File: System/SwatEquipment.ini [hunk 3]
```diff
@@ -6610,6 +6622,7 @@ DeactivatedMesh=StaticMesh'mich_static.mich_static_nvg'
 Weight=0.68
 Bulk=1.4
 
+
```
Comment (T-MaxWiese-T) [inline System/SwatEquipment.ini]: Unnecessary

File: System/SwatEquipment.ini [hunk 4]
```diff
@@ -9390,6 +9403,9 @@ Validity=NETVALID_All
 TeamValidity=TEAMVALID_All
 bSelectable=0
 bSpawnable=True
+TypeOfEquipment=EQUIP_Protection
```
Comment (T-MaxWiese-T) [inline System/SwatEquipment.ini:9405]: Unnecessary

File: System/SwatEquipment.ini (hunk 3 above)
Comment (GuiShing) [reply System/SwatEquipment.ini]: Removed the space.

**Pull request closed without merging**